books/models.py

```diff
@@ -461,8 +461,10 @@ class Book(Page):
     cnx_id = models.CharField(
         max_length=255, help_text="This is used to pull relevant information from CNX.",
         blank=True, null=True)
-    salesforce_abbreviation = models.CharField(max_length=255, blank=True, null=True, verbose_name='Internal Salesforce Name')
-    salesforce_name = models.CharField(max_length=255, blank=True, null=True, verbose_name='Website Visible Salesforce Name')
+    salesforce_abbreviation = models.CharField(max_length=255, blank=True, null=True)
+    salesforce_name = models.CharField(max_length=255, blank=True, null=True)
+    salesforce_book_id = models.CharField(max_length=255, blank=True, null=True,
+                                       verbose_name='Salesforce book id (No tracking if left blank)')
```

Comment:
Maybe a reference to this driving the book selection on the book interest/adoption form would be helpful for future folks?

Reply:
@mwvolo Moved the text to help_text and added info about the forms